cpmpy/expressions/core.py

```diff
@@ -238,6 +238,7 @@ def __mul__(self, other):
             return self
         return Operator("mul", [self, other])
     def __rmul__(self, other):
+
```

Comment:
remove all style-only changes, your commit does not actually change core.py

if you want to do style changes, do it as a separate commit